proto/keyspacepb.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+package keyspacepb;
+
+import "pdpb.proto";
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+// Keyspace provides services to manage keyspaces.
+service Keyspace {
```

Comment:
It seems that service is readonly. I wonder how can we create/update/delete keyspaces?

Reply:
Create/ UpdateState/ UpdateConfig can be done through HTTP API.

Reply:
the client should create a keyspace via HTTP?

Reply:
Target Keyspace should exist prior to client's connection. 
A default keyspace with name `"DEFAULT"` and id `0` is created when KeyspaceManager initialize, if a client (with keyspace enabled) does not provide a keyspace name, if will connect to default keyspace.